Emby.Server.Implementations/Library/PathExtensions.cs

```diff
@@ -39,7 +39,7 @@ public static string GetAttributeValue(this string str, string attrib)
             // for imdbid we also accept pattern matching
             if (string.Equals(attrib, "imdbid", StringComparison.OrdinalIgnoreCase))
             {
-                var m = Regex.Match(str, "tt\\d{7}", RegexOptions.IgnoreCase);
+                var m = Regex.Match(str, "tt\\d{7,}", RegexOptions.IgnoreCase);
```

Comment:
AFAIK the id can only be either 7 or 8 digits
```suggestion
                var m = Regex.Match(str, "tt\\d{7,8}", RegexOptions.IgnoreCase);
```

Reply:
Updated.